### PR TITLE
[UFC-825] subpath support

### PR DESCRIPTION
**Pull request by WimJongeneel**

### CODE REVIEW

File: src/DeepLinkModule/javasource/deeplink/implementation/handler/ResponseHandler.java [hunk 1]
```diff
@@ -135,16 +135,15 @@ public static void serveLogin(IMxRuntimeRequest request, IMxRuntimeResponse resp
 
 			if (continuationURL != null && continuationURL.length() > 0) {
 				loginLocation += URLEncoder.encode(continuationURL, java.nio.charset.StandardCharsets.UTF_8.toString());
-			}
+			}
```
Comment (Robinfr) [inline src/DeepLinkModule/javasource/deeplink/implementation/handler/ResponseHandler.java:138]: There's a random tab here I think?

**Review by Robinfr — CHANGES_REQUESTED** [review]
Only 1 comment.

**Review by Robinfr — APPROVED** [review]

**Comment by wegiswes** [comment]
Hi, 

It looks like IMxRuntimeRequest.getRootUrl() is introduced in Mx10 so running the latest version of this module (10.0.4) in Mx9 results in a java compile error. Please update the required mx version in the marketplace accordingly 😇

**Comment by Robinfr** [comment]
@wegiswes As the docs mention, DeepLink 10 is meant to be used with Mx10+. Please use DeepLink 9 for Mx9.

**Pull request merged**